### ISSUE TITLE
Timezone issues

### ISSUE THREAD

**Issue by luciancrasovan** [taylorhakes/fecha#78]
I have tested the parse method for different time zones and see that 

**parse('1990-01-01', 'YYYY-MM-DD')** 

returns null when in the  Lima/Peru timezone. I have modified the test for the first day of month for 1st of January 1990 and this is the output of the test  

   **x first  day of the month 
     expected 0 to equal 631170000000**
 
I did it on a Ubuntu 18.04 VM but cannot reproduce the error on more negative timezones.

**Comment by taylorhakes** [comment]
Thanks for reporting this issue. I tested myself and you are absolutely right. I don't quite understand why it is happening though. In Peru (only timezone I could see it happen) midnight actually returns 1AM instead of midnight on 1st of January 1990. It does not happen on other years or other timezones.

```js
new Date(1990, 0, 1, 0, 0, 0)  -> new Date(1990, 0, 1, 1, 0, 0)
```

I don't understand why that is happening. I have been trying to investigate without any luck so far. Do you have any idea?

**Comment by taylorhakes** [comment]
Thanks for the bug report! It should be fixed in 4.1. Please let me know if you still have the issue. https://github.com/taylorhakes/fecha/releases/tag/4.1.0
